### ISSUE TITLE
Solarized Dark Theme makes certain command line arguments invisible in PowerShell

### ISSUE THREAD

**Issue by pcaversaccio** [microsoft/vscode#144411]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.65.0
- OS Version: Windows_NT x64 10.0.22000
- PowerShell version: 7.2.1

Steps to Reproduce:

1.  Change theme to "Solarized Dark" in VS Code
2. Open the terminal and type for example: `git commit -s -m "blabla"`
![image](https://user-images.githubusercontent.com/25297591/156791691-80bcab3b-1ac8-4de7-a933-517f00694409.png)

If you highlight the line you will see that actually everything is there:
![image](https://user-images.githubusercontent.com/25297591/156791846-d6c677fd-2e33-429c-9e9a-b74c68469bad.png)

I have tried with other shells (cmd, ubuntu (WSL), debian (WSL), kali-linx (WSL)). There everything is fine. It's just the Powershell that has this bug. This bug appeared when I updated today VS Code to the latest version. In the morning when I worked with the older version everything was fine.

**Comment by ForsakenRei** [comment]
I'm using zsh on my pi4 with latest release and have the same issue with solarized dark.
Everything in solarized light works fine though.

**Comment by Cellane** [comment]
I wonder if something changed with the Solarized Dark theme even in the editor itself. Pretty sure that my code now looks way more grey and less white than before:

![image](https://user-images.githubusercontent.com/171604/157147968-4d032ed1-b0cf-4c45-9851-40cc4e2b6cd8.png)

![image](https://user-images.githubusercontent.com/171604/157147940-d1f9ae52-06d2-4c35-9fad-368b1be0f151.png)

**Comment by Tyriar** [comment]
Can repro, pwsh outputs these as "bright black" (`terminal.ansiBrightBlack`) which is `#002b36`, the same as `editor.background`. A similar thing happens in Windows Terminal:

![image](https://user-images.githubusercontent.com/2193314/157266822-0f7fed30-e81e-4f41-ab7a-281e212d8535.png)

I'm hesitant to change the colors here because solarized uses a fixed palette. Also this is where the recent changes came from https://github.com/microsoft/vscode/pull/142670

You can set `"terminal.integrated.minimumContrastRatio": 4.5` to workaround the problem. @isidorn @daviddossett @meganrogge FYI I'm thinking of setting this to 4.5 by default this iteration, I was originally worried about performance overhead but I've been running it since it was introduced and never noticed any issues.

**Comment by Tyriar** [comment]
Marked as a feature request to remember to add to release notes

**Comment by isidorn** [comment]
Changing the default to be have a good contrast makes good sense to me.
I just do not understand how this could affect performance :) I guess you scan colors in some way?

**Comment by ForsakenRei** [comment]
I didn't get that performance concern as well. And changing contrast works totally fine in my case, though I only changed it to 2 instead of 4.5. Powerline fonts and symbols will have some shade if the contrast ratio is too high.
![1](https://user-images.githubusercontent.com/23041178/157274804-5ab1a9bc-f9c0-44c5-981c-e44b4d521c5c.png)

Here's a screenshot with contrast ration = 2.
![2](https://user-images.githubusercontent.com/23041178/157274743-10dea3d8-26a4-448e-9b08-e786acf462ff.png)

**Comment by Cellane** [comment]
Just to confirm, the contrast ratio fix is only for terminal, it will not the look of the source code, right?
I understand that Solarized is defined with certain fixed colors, but my codebase now looks like it’s all comment and no code, due to being all gray. I suppose there is no other way to go back to the old theme than pull it from Git history and create a custom extension based on the old colors?

**Comment by Tyriar** [comment]
@Cellane yes that's right it this change will only affect the terminal.

**Comment by laggardkernel** [comment]
Why the hell changing the default value for `terminal.integrated.minimumContrastRatio`?! If a terminal color scheme doesn't have enough contrast, it's the problem of this very color scheme. Please, stop messing our terminal colors.
